templates/python/package/client.py.twig

```diff
@@ -123,6 +124,18 @@ class Client:
 
         input = open(file_path, 'rb')
         offset = 0
+        counter = 0
+
+        if upload_id != 'unique()':
+            try:
+                result = self.call('get', path + '/' + upload_id, headers)
+                counter = result['chunksUploaded']
+            except:
+                pass
+
+        if counter > 0:
+            offset = counter * self._chunk_size
+            input.seek(offset)
```

Comment:
Here, if the offset is already > size (i.e file is already uploaded), would this seek cause exception or work without issue?

Reply:
I ran following script:

```
from examples.python.appwrite.client import Client
from examples.python.appwrite.services.storage import Storage

client = Client()

(client
  .set_endpoint('https://matej.appwrite.org/v1') # Your API Endpoint
  .set_project('resumable-upload-deno') # Your project ID
  .set_key('47d0423664d58916714074679911ec31e8b307f70d1e7f853a576f8f7aae6aeaf6e88bbac19d0891045c0b650ce8ae7a0c42ca51a8f7effc0b8edcdc8be81ff85140787e32e8dd71e1fb0787bef37a8b006e3ab07e81d58a0f976977ce0dba6f88935998a71d1e2dcbe729408fba99cc921351ef89afb193a52101f8564a3fa5') # Your secret API key
)

storage = Storage(client)

def ff(t):
    print(t)

storage.create_file('b', 'unique()', './vid.mp4', None, None, ff)
```

Then waited for all 4 chunks to upload, and then ran again with new ID instead of `unique()`. There were no issues.

![CleanShot 2022-03-14 at 09 26 25](https://user-images.githubusercontent.com/19310830/158133374-d0a7b717-af18-4ab3-805a-057df482df35.png)

Reply:
Then it's fine with me 😄